### PR TITLE
MINOR: Fix typos introduced into README.md by PR #12

### PR DESCRIPTION
**Pull request by scholzj**
This PR fixes the typos introduced into README.md by previous commits.

### CODE REVIEW

**Comment by scholzj** [comment]
@tombentley Could you have a look?

File: README.md [hunk 1]
```diff
@@ -73,4 +73,4 @@ To conviniently deploy Kafka Connect to OpenShift a [Template is provided](kafka
 
 ### Deploying to Kubernetes
 
-To conviniently deploy Kafka Connect to Kubernetes use the provided yaml files with the Kafka Connect [deployment](kafka-connect/resources/kafka-connect.yaml) and [service](kafka-connect/resources/kafka-connect-service.yaml). This could be sone suing `kubectl apply -f kafka-connect/resources/kafka-connect.yaml` and `kubectl apply -f kafka-connect/resources/kafka-connect-service.yaml`.
\ No newline at end of file
+To conviniently deploy Kafka Connect to Kubernetes use the provided yaml files with the Kafka Connect [deployment](kafka-connect/resources/kafka-connect.yaml) and [service](kafka-connect/resources/kafka-connect-service.yaml). This could be done using `kubectl apply -f kafka-connect/resources/kafka-connect.yaml` and `kubectl apply -f kafka-connect/resources/kafka-connect-service.yaml`.
```
Comment (tombentley) [inline README.md]: conviniently here too

Comment (scholzj) [reply README.md]: Why do I use words which I cannot write? :-)

Fixed @tombentley

**Comment by tombentley** [comment]
lgtm, but I don't have write access to this repo. @ppatierno can you merge this, if you're also happy with it?

**Comment by ppatierno** [comment]
@tombentley I gave you rights for doing that :-)

**Pull request merged**